lib/bigint_primarykey.rb

```diff
@@ -85,7 +85,7 @@ def install_patches!(adapter)
         pk_module = MysqlBigintPrimaryKey
```

Comment:
Not strictly necessary, but I'd `s/pk_module/primary_key_module/` here and L81,L95